plugins/reports/api/api.js

```diff
@@ -166,11 +179,16 @@ var common = require('../../../api/utils/common.js'),
                         return common.returnMessage(params, 401, 'User does not have right to access this information');
                     }
 
-                    common.db.collection('reports').findOne({_id: common.db.ObjectID(id), user: common.db.ObjectID(params.member._id)}, function(err_update, report) {
+                    const query = {_id: common.db.ObjectID(id)};
+                    if (params.member.global_admin !== true) {
+                        query.user = common.db.ObjectID(params.member._id); 
+                    }
```

Comment:
I guess we don't need this piece of code, as it is in `recordUpdateOrDeleteQuery` function

Reply:
done